### PR TITLE
Fix issue from #22 and fixed bottom selectors from #23

### PR DESCRIPTION
**Pull request by Dancanplay**
Replaced red background to dark with white text, and fixed the bottom selectors:
image

### CODE REVIEW

File: dark.css [hunk 1]
```diff
@@ -1333,3 +1349,21 @@ a.yt-uix-button.yt-uix-button-epic-nav-item, button.yt-uix-button-epic-nav-item,
...
+.yt-picker-content strong {
+	color: #888 !important;
+}
+
+h3.yt {
+	color: #fff !important;
+}
+
+a {
+	color: #fff !important;
```
Comment (NSDex) [inline dark.css]: What is this for?  It's breaking a lot of the other styling (e.g, the headers in the sidebar).  Can you use a more specific selector?

**Comment by NSDex** [comment]
The New badge looks good.  And it's not causing any issue with the Verified badge.

**Comment by Dancanplay** [comment]
It should be good now, check if any issues still occur.

**Comment by NSDex** [comment]
Looks good.  Thanks!

**Pull request merged**